lib/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@synonymdev/react-native-ldk",
   "title": "React Native LDK",
-  "version": "0.0.84",
+  "version": "0.0.87",
```

Comment:
Current version in Bitkit is already `0.0.87`?

> "@synonymdev/react-native-ldk": "0.0.87",

Reply:
npm publish ups the version but I didn't do a follow up PR with new version previously